code/game/machinery/announcement_system.dm

```diff
@@ -88,6 +88,10 @@ GLOBAL_LIST_EMPTY(announcement_systems)
 		message = CompileText(newhead, user, rank)
 	else if(message_type == "ARRIVALS_BROKEN")
 		message = "The arrivals shuttle has been damaged. Docking for repairs..."
+	else if(message_type == "AICRYO" && newheadToggle)
+		message = CompileText("%PERSON has been moved to intelligence storage.", user, rank)
+	else if(message_type == "CRYOSTORAGE")
+		message = CompileText("%PERSON, %RANK has been moved to cryogenic storage.", user, rank)
```

Comment:
I'd just get rid of this. I don't think the issues that lead to cryo storage being removed applies to AIs because they don't have items. This just allows them to leave the round while freeing up the job slot for someone else.

Reply:
Alright but I'm hitting the hay tonight,  will do tomorrow